src/styles/blocks/card.scss

```diff
@@ -0,0 +1,135 @@
+.card {
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  background: #fff;
+
+  transition: transform 0.3s;
+
+  cursor: pointer;
+
+  &__wrapper {
+    display: flex;
+    flex-direction: column;
+    padding: 32px 16px 16px;
+  }
+
+  &__img {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__text-wrapper {
+    margin-top: 40px;
+  }
+
+  &__title {
+    margin: 0;
+
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+
+    color: $color-main;
+
+    transition: color 0.3s;
+  }
+
+  &__subtitle {
+    margin-top: 4px;
+
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
```

Comment:
`font-weight` is 400 by default, you don't need to set it manually.
Fix this issue in all rules.